README.md

```diff
@@ -68,19 +68,33 @@ to New Relic using an [Azure Policy](https://docs.microsoft.com/en-us/azure/azur
 This is the easiest option as it only needs to be setup once and all existing and future
 App Insights resources will be automatically configured to report to New Relic. Keep in
 mind there are costs associated with exporting telemetry data from Azure. You will want
-to monitor integration utilzation to ensure they meet your expectations.
+to monitor integration utilization to ensure they meet your expectations.
 
-To setup this Azure Policy, rrun the below command using the Azure CLI:
+In addition to the Azure CLI, you'll need the `jq` JSON processing tool installed. 
 
-```
-az deployment group create \
-  --name NewRelicLogsPolicy \
-  --resource-group NewRelicLogs \
-  --template-uri https://raw.githubusercontent.com/newrelic/newrelic-azure-log-ingestion/main/templates/azure-policy.json
-```
+To set up this Azure Policy, run the following commands using the Azure CLI:
 
-Replacing, as necessary, the `--resource-group` argument with the ResourceGroup
-that you created during installation.
+    az policy definition create \
+        --name New-Relic-Diagnostic \
+        --mode All \
+        --rules $( jq '.properties.policyRule' < templates/diagnostic-policy.json ) 
```

Comment:
Couldn't these be split into separate JSON files? That appears to be what Azure does in their policy examples.

Also, can these be URLs? That way the user doesn't need to clone the repo to run these commands.

Reply:
I don't know if they can be URLs. The documentation says JSON strings or files, so I'd be surprised if that worked. 

I'll split them into separate files though.

Reply:
[This example](https://github.com/Azure/Community-Policy/tree/master/Policies/App%20Service/deploy-app-service-diag-log-analytics) has this in the README:

```
az policy definition create --name 'deploy-app-service-diag-log-analytics' --display-name 'Deploy App Service Diagnostics Settings' --description 'This policy deploys Diagnostic Settings for App Service to a Log Analytics workspace' --rules 'https://raw.githubusercontent.com/Azure/Community-Policy/master/Policies/App%20Services/deploy-app-service-diag-log-analytics/azurepolicy.rules.json' --params 'https://raw.githubusercontent.com/Azure/Community-Policy/master/Policies/App%20Services/deploy-app-service-diag-log-analytics/azurepolicy.parameters.json' --mode All

az policy assignment create --name <assignmentname> --scope <scope> --policy "deploy-app-service-diag-log-analytics" 

```

So it appears that URLs can be used.

Reply:
Alright, I've updated the README.